pkg/virtual/framework/forwardingregistry/rest.go

```diff
@@ -37,7 +39,7 @@ type StorageWrapper func(schema.GroupResource, *StoreFuncs) *StoreFuncs
 
 // NewStorage returns a REST storage that forwards calls to a dynamic client
 func NewStorage(ctx context.Context, resource schema.GroupVersionResource, apiExportIdentityHash string, kind, listKind schema.GroupVersionKind, strategy customresource.CustomResourceStrategy, categories []string, tableConvertor rest.TableConvertor, replicasPathMapping fieldmanager.ResourcePathMappings,
-	dynamicClusterClient dynamic.ClusterInterface, patchConflictRetryBackoff *wait.Backoff, wrapper StorageWrapper) (mainStorage, statusStorage *StoreFuncs) {
+	dynamicClusterClient dynamic.ClusterInterface, patchConflictRetryBackoff *wait.Backoff, wrapper StorageWrapper, transformers transforming.Transformers) (mainStorage, statusStorage *StoreFuncs) {
```

Comment:
are these transformers more than just a client wrapper? In other words: do we really have to plumb it that deeply into the storage? Wouldn't client composition do too and make it much simpler?

Reply:
Do the changes done in commit 3f0d7ee5e0878187dce7f08113d72eed48d8ba54 correspobd to what you had in mind ?